src/ColorWrapper.cpp

```diff
@@ -0,0 +1,110 @@
+#include <Gosu/Color.hpp>
+#include <Gosu/Color.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+Gosu_Color* Gosu_Color_create(unsigned argb)
+{
+  return reinterpret_cast<Gosu_Color*>( new Gosu::Color(argb) );
+}
+Gosu_Color* Gosu_Color_create_argb(Channel a, Channel r, Channel g, Channel b)
+{
+  return reinterpret_cast<Gosu_Color*>( new Gosu::Color(a, r, g, b) );
+}
+Gosu_Color* Gosu_Color_create_from_hsv(double h, double s, double v)
+{
+  return reinterpret_cast<Gosu_Color*>( &Gosu::Color::from_hsv(h, s, v) );
+}
+Gosu_Color* Gosu_Color_create_from_ahsv(Channel alpha, double h, double s, double v)
+{
+  return reinterpret_cast<Gosu_Color*>( &Gosu::Color::from_ahsv(alpha, h, s, v) );
```

Comment:
This will crash, as `from_ahsv` returns a temporary object, which is deleted as soon as the function returns.